compiler/rustc_codegen_ssa/src/base.rs

```diff
@@ -689,7 +687,7 @@ pub fn codegen_crate<B: ExtraBackendMethods>(
     // This likely is a temporary measure. Once we don't have to support the
     // non-parallel compiler anymore, we can compile CGUs end-to-end in
     // parallel and get rid of the complicated scheduling logic.
-    let mut pre_compiled_cgus = if cfg!(parallel_compiler) {
+    let mut pre_compiled_cgus = if is_parallel() {
```

Comment:
This should do `tcx.sess.threads() > 1` instead of looking up the synchronization mode..

Reply:
Done